bolt/test/Inputs/process-debug-line.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
```

Comment:
Could place this script in `bolt/test` directory (parent one), and link it in `lit.cfg.py` as a tool, similarly with how `link_fdata` is done.